modules/atomic-widgets/assets/js/editor/commands/styles.js

```diff
@@ -43,6 +43,8 @@ export class Styles extends $e.modules.editor.document.CommandHistoryDebounceBas
 		} );
 
 		container.model.set( 'styles', isRedo ? changes.new.styles : changes.old.styles );
+
+		$e.run( 'document/atomic-widgets/styles-updated', { container } );
```

Comment:
Why does this command work with the history but the 'styles' command doesn't?

I think both commands will appear in the history...